libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -317,7 +317,9 @@
             "Inactive": "Inaktiv",
             "Passive": "Passiv",
             "Deleted": "Slettet"
-          }
+          },
+          "statusChangeWarningActionLabel": "Ja",
```

Comment:
suggestion: there are "Yes"/"No" translations at the top of the file. If we know that `statusChangeWarningActionLabel` is going to always be "Yes", then we can use the global translation.